Lib/test/test_syntax.py

```diff
@@ -972,6 +972,9 @@ def func2():
 """
         self._check_error(code, "invalid syntax")
 
+    def test_unexpected_line_continuation(self):
+        self._check_error('A.\u018a\\ ', "unexpected character after line continuation character")
```

Comment:
I would test two cases: `\\\n` and `\\` followed by non-`\n` character. They are two slightly different errors.

Instead `\u018a` you can use any non-ascii character.